cmd/clusterctl/cmd/init.go

```diff
@@ -83,7 +83,7 @@ var initCmd = &cobra.Command{
 
 func init() {
 	initCmd.Flags().StringVarP(&io.kubeconfig, "kubeconfig", "", "", "Path to the kubeconfig file to use for accessing the management cluster. If empty, default rules for kubeconfig discovery will be used")
-	initCmd.Flags().StringVarP(&io.coreProvider, "core", "", "", "Infrastructure providers to add to the management cluster. By default (empty), the cluster-api core provider is installed on the first init")
+	initCmd.Flags().StringVarP(&io.coreProvider, "core", "", "", "Core providers to add to the management cluster. By default (empty), the cluster-api core provider is installed on the first init")
```

Comment:
I assume we don't support multiple core providers. We probably need some slightly different language here.

Reply:
:D left a comment thinking the same

Reply:
since it appears we can use the flag like `--core cluster-api:v0.3.0` perhaps this should read:

Core provider version (by tag) to add to the management cluster. By default (empty), the cluster-api core provider's latest release is used."

Thoughts?

Reply:
That sounds good to me!

Reply:
+1 for me

Reply:
updated the doc help command based upon what was discussed above. Also added "version and tag" info to the bootstrap and infrastructure commands because they support the same format as core.